### PR TITLE
[Feature]: SSH Verbosity Levels

### PR DESCRIPTION
**Pull request by mitchtreece**
Added the ability to specify an optional SSH verbosity level in the shipit configuration file.

```node
{
    ...

    // 0 (none), 1 (-v), 2 (-vv), 3 (-vvv)
    // Defaults to `none`
    verboseSSHLevel: 1
}
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <h1 align="center">
-  <img src="https://raw.githubusercontent.com/shipitjs/shipit/master/ressources/shipit-logo.png" alt="Shipit" title="Shipit" width="500">
+  <img src="./resources/shipit-logo.png" alt="Shipit" title="Shipit" width="500">
```
Comment (neoziro) [inline README.md]: Broken in diff.

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@ node_modules/
 packages/*/lib
 coverage/
 .eslintcache
+.DS_Store
```
Comment (neoziro) [inline .gitignore]: It should be in your global `.gitignore`, please remove it.

File: package.json [hunk 3]
```diff
@@ -33,5 +32,8 @@
     "std-mocks": "^1.0.1",
     "string-length": "^2.0.0"
   },
-  "workspaces": ["packages/*"]
+  "workspaces": [
+    "packages/*"
+  ],
+  "dependencies": {}
```
Comment (neoziro) [inline package.json]: Unnecessary changes please, remove it.

File: packages/shipit-cli/src/Shipit.js [hunk 4]
```diff
@@ -158,6 +158,7 @@ class Shipit extends Orchestrator {
       ...this.options,
       key: this.config.key,
       strict: this.config.strict,
+      verbosityLevel: Object.is(this.config.verboseSSHLevel, undefined) ? 0 : this.config.verboseSSHLevel
```
Comment (neoziro) [inline packages/shipit-cli/src/Shipit.js]: I think `Object.is` is not compatible with old version, `===` will be fine.

**Comment by neoziro** [comment]
Thanks for your PR, just a few minor changes please.

**Comment by neoziro** [comment]
@mitchtreece good! A last thing, can you please update all readme to reflect the change?

**Comment by mitchtreece** [comment]
@neoziro You got it. Updated! 🎉

**Comment by neoziro** [comment]
I think you missed this one https://github.com/shipitjs/shipit/blob/master/packages/ssh-pool/README.md

**Comment by mitchtreece** [comment]
@neoziro Oops. Should be good now!

**Comment by neoziro** [comment]
Thanks!
